cmd/krel/cmd/gcbmgr.go

```diff
@@ -230,24 +239,27 @@ func setGCBSubstitutions() (map[string]string, error) {
 	gcbSubs["RELEASE_TOOL_REPO"] = releaseToolRepo
 	gcbSubs["RELEASE_TOOL_BRANCH"] = releaseToolBranch
 
-	// TODO: Need to find out if command.Execute supports capturing the command output
-	gcpUser := "FAKEUSER"
-	//nolint
-	/*
-		gcpUser := command.Execute(
-			"gcloud",
-			"auth",
-			"list",
-			"--filter=status:ACTIVE",
-			`--format="value(account)"`,
-		)
-		if gcpUser != nil {
-			return nil, gcpUserErr
-		}
-	*/
```

Comment:
Definitely could be changed later, but how would you feel about moving some of this logic to pkg/gcp for reuse?